package.json

```diff
@@ -9,9 +9,9 @@
   "private": false,
   "scripts": {
     "build": "tsc -p ./tsconfig.lib.json",
-    "prepare": "npm run build",
-    "test": "ts-mocha -p ./tsconfig.test.json",
-    "test:unit": "yarn run test src/**/*.spec.ts",
+    "prepare": "yarn build",
+    "test": "yarn jest --detectOpenHandles",
+    "test:ci": "yarn jest --runInBand --detectOpenHandles",
```

Comment:
Maybe we can use here `yarn test ---runInBand`?

Reply:
Fair point!